src/Core/Manifest/Module.php

```diff
@@ -247,3 +247,14 @@ public function hasResource($path)
             ->exists();
     }
 }
+
+/**
+ * Scope isolated require - prevents access to $this, and prevents module _config.php
+ * files potentially leaking variables
+ *
+ * @param string $file
+ */
+function requireFile($file)
+{
+    require_once $file;
```

Comment:
I was doing a bit of light reading, and have found that `$file` leaks into `_config.php` this way. The way suggested was to use `require func_get_arg(0);`

This is quite minor I guess because `$file` isn't such a big deal to leak in to.

Reply:
I like it, if we’re overkilling this then let’s do it properly 😉